main.nf

```diff
@@ -1095,6 +1095,8 @@ process multiqc {
     output:
     file "*multiqc_report.html" into multiqc_report
     file "*_data"
+    file "multiqc_plots/*"
+    file "software_versions.csv"
```

Comment:
Almost! This line needs to go in the software versions process, not MultiQC. And you'll need to add `publishDir` logic to that process too..

Reply:
Thank you for the tip @ewels ! I had put the `software_versions.csv` in the `pipeline_info` folder, it made sense to me - but I don't know if it's coherent with the rest of the `main` structure.